preload.js

```diff
@@ -113,8 +113,12 @@
   const readAttachmentData = Attachments.readData(attachmentsPath);
   const writeAttachmentData = Attachments.writeData(attachmentsPath);
 
+  const { IdleDetector} = require('./js/modules/idle_detector');
```

Comment:
This is pretty awkward. I can see why we might want named exports, like for future-proofing as we add more things to files. But perhaps a better approach might be to choose a parent namespace of some kind, and put it inside that. Like `window.Signal.OS` does.